packages/unit-vitest/src/templates/base/src/components/___examples__/___tests__/ExampleComponent.vitest.test.ts

```diff
@@ -0,0 +1,38 @@
+import { installQuasar } from '@quasar/quasar-app-extension-testing-unit-vitest';
+import { mount } from '@vue/test-utils';
+import { describe, expect, it } from 'vitest';
```

Comment:
I switched to use ESM imports, instead of relying on globals, to avoid conflicts with Cypress globals